trin-storage/src/versioned/id_indexed_v1/sql.rs

```diff
@@ -75,7 +75,7 @@ pub fn delete_farthest(content_type: &ContentType) -> String {
             ORDER BY distance_short DESC
             LIMIT :limit
         )
-        RETURNING content_size",
+        RETURNING content_key, content_value",
```

Comment:
nit: I would still return `content_size` as well. This should save us some calculations and make sure we are using the right value.